packages/pwa-kit-create-app/assets/pwa/api.config.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+exports.template = ({commerceApi, einsteinApi}) => `
+export const commerceAPIConfig = {
```

Comment:
Nit: Since we are using literal string, cutting a line will create a blank line in the file. Seems a bit odd, eslint will complain if it is turned on
![image](https://user-images.githubusercontent.com/52219283/145897568-0056e0d1-34da-4309-8676-2e09b3af9a7d.png)